provider/aws/formation/resource/redis.json.tmpl

```diff
@@ -1,5 +1,8 @@
 {
   "AWSTemplateFormatVersion": "2010-09-09",
+  "Conditions": {
+    "IsEncrypted": { "Fn::Equals": [{ "Ref": "Encrypted" }, "true"] }
```

Comment:
Also please call this condition just `Encrypted` for consistency with the other CloudFormation stacks

Reply:
Sure! I wasn't sure if it'd conflict but no problem